packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -184,12 +184,11 @@ export const InputBaseInput = styled('input', {
     display: 'block',
     // Make the flex item shrink with Firefox
     minWidth: 0,
-    width: '100%', // Fix IE11 width issue
```

Comment:
Introduced in https://github.com/mui/material-ui/pull/12975/files#diff-a5f495d99c59823944e8c99682e25d509e79d427ab0442c71589da33e6f350f6R78

Tested it in Chrome, Firefox, and Safari. Screenshot in Chrome:

<img width="246" alt="Screenshot 2024-05-23 at 16 43 04" src="https://github.com/mui/material-ui/assets/16889233/ae010490-166b-4899-ad0f-2e8679a7083a">